src/misc/ErrorsPanel/ErrorsPanel.js

```diff
@@ -43,10 +43,14 @@ export const ErrorsPanel = (props) => {
               </Typography>
             </Box>
             <Box flexGrow={1}>
-              <Typography className={classes.errorSummary}>{error.summary}</Typography>
+              <Typography dataclassName={classes.errorSummary} data-cy={'error-summary'}>
```

Comment:
You changed `className` to `dataclassName`, this will break the style of these elements :sweat_smile:

Reply:
It's a mess...